src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.bigSearchBar {
+  position: relative;
+  margin: 20px 8px;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.smallSearchBar {
+  position: relative;
+  margin: 20px 8px;
+  height: 42px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.bigSearchInput {
+  font-family: Avenir, sans-serif;
```

Comment:
move fonts to html tag, they will be inherited for all page,
and then you can remove them from every class style
it wold be inherited in almost all tags and classes, if not - write font-family: inherit

Reply:
Thanks! I will move.